src/ml_gym/frontend/dashboard/src/routes/flow_board.tsx

```diff
@@ -0,0 +1,122 @@
+import { AgGridReact } from "ag-grid-react";
+import React, { Component } from "react";
+import type { RootState } from '../app/store';
+import 'ag-grid-community/styles/ag-grid.css'; // Core grid CSS, always needed
+import 'ag-grid-community/styles/ag-theme-alpine.css'; // Optional theme CSS
+import { jobStatusRowsSelector } from '../features/jobsStatus/jobsStatusSlice'
+import { connect } from "react-redux";
+
+import { JobStatusRowType, ModelStatusRowType } from '../app/datatypes'
```

Comment:
Codacy has a fix for the issue: [Replace `'../app/datatypes'` with `"../app/datatypes";`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
import { JobStatusRowType, ModelStatusRowType } from "../app/datatypes";
```